### DISCUSSION TITLE
Contact list project ideas

### DISCUSSION THREAD

**Discussion by Anurag-Negi28** [Anurag-Negi28/contact_list#2] [Ideas]
Hey Hardik we should work on use case of our project.

**Comment by Anurag-Negi28** [comment]
![WhatsApp Image 2023-09-20 at 19 11 22](https://github.com/Anurag-Negi28/contact_list/assets/123005508/b10428b4-15ca-447f-9324-6992af6def90)
Check this

**Reply by HardikYadav99** [reply]
Hey Anurag we can add a feature for profile photo of the user as well.
The use case is good maybe we can make it digitally using canvas or milanote.

**Reply by Anurag-Negi28** [reply]
Yeah Hardik 
currently working on it, I will be sharing the use case as soon as I finish it.

**Reply by Anurag-Negi28** [reply]
Hey Hardik you have to be active on this channel
Just replying to my comment took you one week with that speed we will not be able to deliver the project

**Comment by Anurag-Negi28** [comment]
Check this use case - I have applied visual representation as a part of Kanban methodology in it.
![canvas_contact-list-use-case-230921_1630](https://github.com/Anurag-Negi28/contact_list/assets/123005508/e84b5932-cb82-4f64-9368-85ce3548d1ad)

**Reply by Anurag-Negi28** [reply]
I have also made another use case as a user and admin point of view.
![image](https://github.com/Anurag-Negi28/contact_list/assets/123005508/fac3f0b5-e5d8-4a2d-9632-45afd685e8f8)

**Comment by Anurag-Negi28** [comment]
I have also made flowchart of the same, you can review that
![diagram-export-9_28_2023, 9_26_22 PM](https://github.com/Anurag-Negi28/contact_list/assets/123005508/2f6f53d2-cbc5-4f79-b0d3-5c5fab24374b)

**Comment by HardikYadav99** [comment]
I have alternative use case have a look on it, it is quite simpler I guess.
![Use Case Diagram Team Whiteboard in Yellow Pink Green Spaced Color Blocks Style 2](https://github.com/Anurag-Negi28/contact_list/assets/143898154/d4a42fc8-2cbf-4cd0-a490-bcaad3a4d11d)

**Reply by Anurag-Negi28** [reply]
Yeah it is also good
I guess we can show all of the outcomes  to Sir as a part of Kanban methodology

**Reply by Anurag-Negi28** [reply]
You can commit this on the use case folder in the repository

**Comment by Anurag-Negi28** [comment]
Hey Hardik as we are completed with the use case we should now focus on the coding part.
